engine/impl/Playlist.php

```diff
@@ -162,39 +164,86 @@ public function updatePlaylist($playlist, $date, $time, $description, $airname)
     }
 
     public function getTrack($id) {
-        $query = "SELECT tag, artist, track, album, label, id FROM tracks " .
+        $query = "SELECT created, tag, artist, track, album, label, id FROM tracks " .
```

Comment:
New column 'created' needs to go to the end of the existing SELECT columns.  There may be some places in the code that reference columns by ordinal.  By putting it last, you ensure you don't break existing ordinal references.